db/schema.rb

```diff
@@ -137,7 +137,9 @@
     t.string "email"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "uid"
     t.index ["personable_type", "personable_id"], name: "index_people_on_personable"
+    t.index ["uid"], name: "index_people_on_uid"
```

Comment:
People are unregistered humans, Users are registered humans. Do you want this `uid` on the User table or Person? 

To be honest, introducing Person was a bit of a premature abstraction on my part. I was modeling the registration off the Basecamp pattern where deactivated users convert to a Tombstone object, invited users are People but no User or Tombstone.

Reply:
About the premature abstraction: 

I think the key question is how you would like to envision registration to happen in the long term. Out in the open for everyone like it is now?

Or is it a more private service, where an initial user gets created and others get invited.

Reply:
I am going to move the uid to the User model